sdk/storage/azure-storage-file-share/pom.xml

```diff
@@ -60,6 +60,8 @@
 
     <javadoc.excludePackageNames>com.azure.json,com.azure.xml,com.azure.core.*</javadoc.excludePackageNames>
     <javadoc.sourcepath>${basedir}/src/main/java:${basedir}/../../core/azure-core/src/main/java:${basedir}/../../serialization/azure-json/src/main/java:${basedir}/../../serialization/azure-xml/src/main/java</javadoc.sourcepath>
+
+    <revapi.differences.file>revapi-differences.json</revapi.differences.file>
```

Comment:
nit: Should we call this the revapi-suppressions.json?

Reply:
I went with `revapi-differences.json` as that is what RevApi calls them, but I'm not opposed to `revapi-suppressions.json`